tests/test_make.py

```diff
@@ -246,26 +246,30 @@ class C(object):
         assert sentinel == getattr(C, method_name)
 
     @pytest.mark.skipif(not PY3, reason="__qualname__ is PY3-only.")
-    def test_repr_qualname(self):
+    @pytest.mark.parametrize('slots_outer', [True, False])
```

Comment:
oh poor man’s hypothesis, I see. :)   hm I think I prefer a more explicit version though b/c I’ll quickly forget what this means.  But you can totally precompute the cartesian product somewhere.